torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -567,6 +567,19 @@ RegisterOperators reg(
            return 0;
          },
          aliasAnalysisFromSchema()),
+     Operator(
+         "prim::grad(Tensor a) -> Tensor",
```

Comment:
Alias information is wrong here `(Tensor a) -> Tensor(*)`. It is similar to looking up a tensor from a list.